spring-ldap-2.3.3.RELEASE/pom.xml

```diff
@@ -46,6 +46,7 @@
         <pkgGroupId>org.springframework.ldap</pkgGroupId>
         <pkgVersion>2.3.3.RELEASE</pkgVersion>
         <servicemix.osgi.export.pkg>
+        	org.springframework,
```

Comment:
It would be great to have the split-package flag here, else we will have "conflict" with regular spring bundles.

Reply:
You are right, I have added split-package directive